remill/Arch/AArch64/Arch.cpp

```diff
@@ -737,16 +745,26 @@ static bool MostSignificantSetBit(uint64_t val, uint64_t *highest_out) {
     }
   }
   return found;
+#endif
 }
 
 static bool LeastSignificantSetBit(uint64_t val, uint64_t *highest_out) {
+#if __has_builtin(__builtin_ctzll)
+  if (val) {
+    *highest_out = __builtin_ctzll(val);
```

Comment:
Given that above you have a `sizeof(unsigned long long)`, which is there to kind of handle the case that it's possibly a different size than `uint64_t`, then I think you should add in a

```c++
static_assert(sizeof(unsigned long long) >= sizeof(uint64_t),
              "Invalid use of `__builtin_clzll` on a `uint64_t`.");
```